### ISSUE TITLE
`jekyll new` doesn't tell you to enter the directory

### ISSUE THREAD

**Issue by AnsonYao** [jekyll/jekyll#4834]
###### What version of Jekyll are you using (`jekyll -v`)?

3.1.3

###### What operating system are you using?

Windows 10

###### What did you do?
(Please include the content causing the issue, any relevant configuration settings, and the command you ran)

input:
```
jekyll new abc
jekyll build
```

response: 

`Liquid Exception: Included file '_includes/icon-github.html' not found in abc/about.md`

###### What did you expect to see?
I have realized I need to cd to myblog in order to build. But It took me quite a while trying to fix the about.md and I hope the response can be more helpful in this case.

**Comment by ChaosCat** [comment]
@bigface A **symbolic link** (symlink) is just a file that contains reference to other file/s, nothing wrong with that error message the full one is this:
```yml
Configuration file: none
            Source: /home/elian/Documents/Development/Jekyll Sites
       Destination: /home/elian/Documents/Development/Jekyll Sites/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
     Build Warning: Layout 'post' requested in sandbox/_posts/2016-04-25-welcome-to-jekyll.markdown does not exist.
Liquid Exception: Included file '_includes/icon-github.html' not found in sandbox/about.md
```
As you can see the error simply says that when it checked the sub folder of the root of the project (which is in this case the root of the project itself), it found a markdown file, with the property of `layout: post` and it also had a request from `about.md` for a file in the `_includes` subdirectory of the folder where `about.md` is, but it didn't check deeper since the build command is designed *(I assume)* to check only to one folder depth for layouts and plugins, includes etc... thus, it didn't find the file names that the markdown files requested. **Which makes this error completely fine.**

**Comment by parkr** [comment]
Hey @bigface! I agree, the error message you saw isn't that useful. When you execute `jekyll new`, it tells you where the blog is and it's common for `new` commands to produce something you have to go into, like `mkdir myblog` requires a `cd myblog` for `pwd` to reflect `myblog`.

What would you change about this message? We can't allude to UNIX commands as many Jekyll users use Windows. The message is:

```text
~$ jekyll new myblog
New jekyll site installed in /Users/parkr/myblog.
```

How would you like to see that improved to be more direct?

**Comment by bigface** [comment]
@ChaosCat, Thank you for explaining the error. I have figured out why the error looks like this. I suggest the error message can be improved (if possible). 

@parkr, I agree that I made a mistake and forgot to cd to my folder. 
I am not very sure about how Jekyll works. Let me assume that Jekyll will first try to find some files in the present working directory and then try to process those files. The current message is 

`Liquid Exception: Included file '_includes/icon-github.html' not found in abc/about.md`

It suggested that JekyII has figured out about.md is in the abc folder. Then, I will assume that JekyII knows it should search my abc folder for files it need, which turns out to be not true. 

I hope the message can be clear that Jekyll did not find _config.yml (or anything in the root directory). Then users will be able to figure out the error is about the directory. If it fails to build the project without cd to folder, it should fail at the beginning and consistently with all files, not in the middle way.
